README.md

```diff
@@ -462,6 +462,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Jersey](https://jersey.java.net/) - JAX-RS reference implementation.
 * [RESTEasy](http://resteasy.jboss.org/) - Fully certified and portable implementation of the JAX-RS specification.
 * [RestExpress](https://github.com/RestExpress/RestExpress) - Thin wrapper on the JBOSS Netty HTTP stack to provide scaling and performance.
+* [Restlet Framework](http://restlet.com/products/restlet-framework/) - The flexible and powerful pioneer REST Web API framework for Java.
```

Comment:
Please remove the article from the beginning for consistency with other entries. Also remove the "for Java" it is implied.

What does "pioneer" mean?

Why is "Web" capitalized?